Client/mods/deathmatch/logic/CResource.cpp

```diff
@@ -267,14 +267,14 @@ void CResource::Load()
     }
 
     // Load the no cache scripts first
-    for (std::list<SNoClientCacheScript>::iterator iter = m_NoClientCacheScriptList.begin(); iter != m_NoClientCacheScriptList.end(); ++iter)
-    {
+    while (!m_NoClientCacheScriptList.empty()) {
```

Comment:
simple and readable variant 
`for (SNoClientCacheScript item : m_NoClientCacheScriptList)`